index.html

```diff
@@ -10696,19 +10697,19 @@ <h2 id="AudioWorklet-Examples">
             </dt>
             <dd>
               <p>
-                This method is DEPRECATED. It is equivalent to setting
-                <code>positionX.value</code>, <code>positionY.value</code>, and
-                <code>positionZ.value</code> attribute directly with the
-                <code>x</code>, <code>y</code> and <code>z</code> parameters,
+                This method is DEPRECATED. It is equivalent to setting <a href=
+                "#dom-pannernode-positionx">positionX.value</a>, <a href=
```

Comment:
I think links should be `<a data-link-for="PannerNode">positionX</a><code>.value</code>`.